docs/user/dql/functions.rst

```diff
@@ -2459,6 +2459,27 @@ Example::
     | 733687                       |
     +------------------------------+
 
+TO_SECONDS
+----------
+
+Description
+>>>>>>>>>>>
+
+Usage: to_seconds(date) returns the seconds number (the number of days since year 0) of the given date. Returns NULL if date is invalid.
```

Comment:
A typo?
```suggestion
Usage: to_seconds(date) returns the number of seconds since the year 0 of the given value. Returns NULL if value is invalid.
```

Reply:
Addressed in https://github.com/Bit-Quill/opensearch-project-sql/commit/9a2377a5b8ee4bcf5df64a5cea5bf20bd069d32b